prow/bugzilla/types.go

```diff
@@ -107,6 +107,8 @@ type Bug struct {
 	Version []string `json:"version,omitempty"`
 	// Whiteboard is he value of the "status whiteboard" field on the bug.
 	Whiteboard string `json:"whiteboard,omitempty"`
+	// PRs holds the links to the pull requests associated with the bug.
+	PRs []ExternalBug
```

Comment:
Also I wanted to call attention to this change. Since this is an incremental change I dont think it would break anything, but just wanted someones go ahead

Reply:
It should be fine, but please add a `json:"fieldname,omitempty" marker